.github/workflows/metal_acctest.yml

```diff
@@ -14,12 +14,10 @@ name: Metal Acceptance Tests
 on:
   pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths-ignore:
-      - 'LICENSE'
-      - '**.md'
-      - 'website/**'
-      - 'docs/**'
-      - '.github/ISSUE_TEMPLATE/**'
+    paths:
+      - 'go.mod'
+      - 'go.som'
+      - '**/*.go'
```

Comment:
Is there an option to combine this with a path-ignore of fabric/ne/...? If a PR only modified an existing NE resource, we could skip Metal tests.

Reply:
I think it's safer to assume the code might be shared